src/components/ui/dropdown/dropdown-renderer.js

```diff
@@ -169,23 +169,6 @@ const DropdownRenderer = ({
   );
 };
 
-DropdownRenderer.defaultProps = {
-  children: null,
-  defaultText: 'Please select...',
-  disabled: false,
```

Comment:
You might want to add the `false / true` values as defaults in the props?

Reply:
ah yeah true 👍, dont know why the tests didnt catch this